app/helpers/visit_helper.rb

```diff
@@ -46,7 +46,7 @@ def prison_slot_anomalies
   end
 
   def prison_address(glue='<br>')
-    prison_data['address'].join(glue).html_safe
+    safe_join(prison_data['address'], glue.html_safe)
```

Comment:
No: even `content_tag` won't work (it will generate `<br></br>`). I would suggest a small change, though: to move the `.html_safe` to the default parameter, so that anyone overriding `glue` has to take responsibility for its safety.